k0776243/Midterm.py

```diff
@@ -0,0 +1,80 @@
+# Skeletons invade pirate ship.  Hero is stuck in bottom of ship and has to escape, get off the boat and to the island.  
+from sys import exit
+from random import randint
+
+class Ship(object):
+    
+    def enter(self)
+        print "You're stuck in the bottom of the ship and it's filling rapidly with water."
+        print "Choose between: a crowbar, the trapdoor key, or broccoli."
+    
+    action = raw_input("> ")
+    
+    if action == "crowbar":
+        print "You bash the crowbar into the overhead trapdoor."
+        print "You pry it open and a skeleton sees you and stabs you."
+        print "You die.  You have two lives left."
+        
+    elif action == "trapdoor key":
+        print "You find the key and unlock the trapdoor."
+        print "Once you get out in the open, you run for it, promptly fall in a puddle and off the ship."
+        print "There's a shark in the water.  You die.  You have two lives left."
+        
+    elif action == "broccoli":
+        print "You find stale broccoli."
+        print "You make friends with the shark through a window and he bashes it open.  You're free."
+        
+
+
+class Water(object):
+    
+    def __init__(self, scene_map):
```

Comment:
I think that instead of `__init__(self, scene_map):` you want `enter():` like the other _Scene_ objects?